cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/stream/FileStreamAdmin.java

```diff
@@ -288,58 +305,52 @@ public StreamConfig call() throws IOException {
 
   @Override
   public StreamProperties getProperties(Id.Stream streamId) throws Exception {
+    // User should have read access on the stream to read its properties
+    ensureAccess(streamId.toEntityId(), Action.READ);
```

Comment:
where is ``getProperties`` used? Is it as part of a "view" request for a stream on the UI/CLI? If so, it should be ok to have "any" privileges on the stream to be able to perform this operation, right?